virttest/libvirt_xml/vm_xml.py

```diff
@@ -1562,7 +1562,7 @@ def get_blkdevio_params(vm_name, options="", virsh_instance=base.virsh):
 
     @staticmethod
     def set_memoryBacking_tag(vm_name, hpgs=True, nosp=False, locked=False,
-                              virsh_instance=base.virsh):
+                              access_mode=None, virsh_instance=base.virsh):
```

Comment:
It is better to always add new arguments at last for better backward compatibility or to change all the users accordingly.

Reply:
Updated.